dist/images/ovnkube.sh

```diff
@@ -754,7 +754,7 @@ echo "================== ovnkube.sh --- version: ${ovnkube_version} ============
   display_version
 
   # get api server
-  k8s_server=$(KUBECONFIG=${KUBECONFIG:-"/etc/origin/node/node.kubeconfig"} kubectl config view --minify | awk '/server:/{ print $2 }')
+  k8s_server="https://${KUBERNETES_PORT_443_TCP_ADDR}:${KUBERNETES_SERVICE_PORT_HTTPS}"
   k8s_apiserver=${K8S_APISERVER:-${k8s_server}}
```

Comment:
Note that you will need the K8s API Server's Node IP here and not the Service IP. On the Kuberenetes Node, we would have not yet setup the L3 gateway and we can't translate 10.96.0.1 to K8s API Server's IP address.